samples/python/hello_classification/hello_classification.py

```diff
@@ -57,10 +57,18 @@ def main():
     # - precision of tensor is supposed to be 'u8'
     # - layout of data is 'NHWC'
     # - set static spatial dimensions to input tensor to resize from
-    ppp.input().tensor() \
-        .set_element_type(Type.u8) \
-        .set_layout(Layout('NHWC')) \
-        .set_spatial_static_shape(h, w)  # noqa: ECE001, N400
+    if (device_name != 'MYRIAD'):
+        ppp.input().tensor() \
+            .set_element_type(Type.u8) \
+            .set_layout(Layout('NHWC')) \
+            .set_spatial_static_shape(h, w)  # noqa: ECE001, N400
+    else:
```

Comment:
I think you should use variable for type because layout and shape are equal in both cases. Please create like something: if (device_name != 'MYRIAD')  element_type = Type.u8  else element_type = Type.f32 and .set_element_type(element_type)

Reply:
Sure, thank you. Done.